### PR TITLE
tflite.cc improvements

### PR DESCRIPTION
**Pull request by alanvgreen**
Several quality-of-life improvements for tflite.cc

### CODE REVIEW

File: common/src/tflite.cc [hunk 1]
```diff
@@ -246,12 +258,13 @@ void tflite_classify() {
   if (kTfLiteOk != interpreter->Invoke()) {
     puts("Invoke failed.");
   }
+  printf("\n");
   uint32_t end = perf_get_mcycle();
 #ifndef NPROFILE
-  profiler->Log();
-#endif
+  profiler->LogCsv();
```
Comment (danc86) [inline common/src/tflite.cc:264]: I sent this one last night in https://github.com/google/CFU-Playground/pull/158 :-)

**Review by danc86 — APPROVED** [review]

**Comment by tcal-x** [comment]
@avg, the test failure can be reproduced if you go to proj/mnv2_first/, run "`make renode`", and then try to run the mnv2_first golden tests.    I see some of the new progress bar (`.....`), but then it seems to hang.

**Comment by tcal-x** [comment]
> @AvG, the test failure can be reproduced if you go to proj/mnv2_first/, run "`make renode`", and then try to run the mnv2_first golden tests. I see some of the new progress bar (`.....`), but then it seems to hang.

@avg, actually I'm not sure what's going on, since the Renode seems to hang at that point (when I run the test as described) even on current main.

**Comment by tcal-x** [comment]
@alanvgreen , I have some of it figured out.   The right command to run it yourself is:
```
make SW_ONLY=1 clean renode
```

Then, with these changes, the mnv2_first golden tests run (they don't hang), but they do fail due to different output values.   Note that we're not using the gateware, we're using the C emulation for the CFU here.   It does pass on current main.

File: common/src/tflite.cc [hunk 2]
```diff
@@ -181,33 +193,33 @@ void tflite_load_model(const unsigned char* model_data,
...
   memcpy(input->data.int8, data, input->bytes);
-  printf("Copied %d bytes at 0x%p\n", input->bytes, input->data.int8);
+  printf("Copied %d bytes at %p\n", input->bytes, input->data.int8);
 }
 
 void tflite_set_input_unsigned(const unsigned char* data) {
   auto input = interpreter->input(0);
   for (size_t i = 0; i < input->bytes; i++) {
-    input->data.int8[i] = (int)data[i] - 127;
+    input->data.int8[i] = static_cast<int>(data[i]) - 128;
```
Comment (tcal-x) [inline common/src/tflite.cc:214]: @alanvgreen  Do the test golden output values need updating for this change?

**Comment by alanvgreen** [comment]
Pushed a new version which updates the mnv2 golden tests to account for the fix to tflite_set_input_unsigned() function.

**Pull request merged**